src/scss/index.scss

```diff
@@ -1 +1,154 @@
+$primary: #67927a;
+$secondary: #eae6e5;
+$dark-color: #000;
+$secondary-light: #eae6e5c4;
+$primary-light: #67927ae9;
+$box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.16);
 
+p {
+  color: $primary;
+}
+
+.btn {
+  box-shadow: $box-shadow;
+  text-decoration: none;
+}
+
+/* nav bottom */
+.navbar-bottom {
+  background-color: $primary-light;
+  box-shadow: $box-shadow;
+  border-radius: 60px 60px 0px 0px;
+
+  i.fas {
+    color: $secondary;
+  }
+}
+
+/* body styles */
+body {
+  background-color: $secondary;
+}
+
+.body-login {
+  //image from: https://pixabay.com/illustrations/flowers-wildflowers-bloom-botanical-8100386/
+  background-image: url("/images/flowers-8100386_1280.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+/* Authentication page */
+
+.navbar-custom {
+  background-color: $primary-light;
+  border-radius: 0px 0px 50px 50px;
+  box-shadow: $box-shadow;
+}
+
+.logo {
+  width: 100%; /* Adjust as needed */
+  max-width: 400px; /* Maximum width for the logo */
+  display: block;
+  margin: 0 auto;
+}
+
+/* Log in form */
+.form-container {
```

Comment:
There are too many custom styles here, background colours, border radius, padding, flex, alignment can all be done with Bootstrap styles!

Reply:
started all over again: thanks, I have misunderstood bootstrap and am now deleting custom styling and replacing it with bootstrap styles